### PR TITLE
Add geoloc with react

### PR DESCRIPTION
**Pull request by sproj**
Adds map with geolocation control

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+import mapboxgl from 'mapbox-gl';
+
+mapboxgl.accessToken = 'pk.eyJ1Ijoic3Byb2oiLCJhIjoiY2s5YnI5ZXVmMDc0OTNvbnk0eTN6enYyayJ9.Mewt68SdJMZuqL4aS1sByg'
```
Comment (yingw787) [inline src/index.tsx]: Is this from `.env`?

File: src/serviceWorker.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
+// This optional code is used to register a service worker.
```
Comment (yingw787) [inline src/serviceWorker.ts:1]: Do we need service workers? I'm not familiar with service workers, but my understanding is they provide offline-first access. If my understanding is correct, how useful might that be if we're dynamically loading vector tiles via API?

**Comment by yingw787** [comment]
May want to strip out unnecessary files (logo.svg for example), not sure if `.env` should be committed to version control (may want to rotate your keys), and would be great to add PULL_REQUEST.md to `.github/templates` a la https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

File: src/index.tsx (hunk 1 above)
Comment (sproj) [reply src/index.tsx]: it is, removed from here

File: src/serviceWorker.ts (hunk 2 above)
Comment (sproj) [reply src/serviceWorker.ts:1]: Don't expect we'll be using service workers, this all comes with CRA. Fine with leaving it in if you are.

File: src/index.tsx (hunk 1 above)
Comment (yingw787) [reply src/index.tsx]: :+1:

File: src/serviceWorker.ts (hunk 2 above)
Comment (yingw787) [reply src/serviceWorker.ts:1]: I'm fine with it. I don't think it'll impact number of users or time to ship over the long term.

I think getting a testing / staging pipeline up and running is really important for figuring out whether something works and minimizing time to ship, and I think generally having smaller diffs (among other things like linting) makes it easier for code reviewers to see bugs.

**Comment by yingw787** [comment]
Discussed, LGTM

**Review by yingw787 — APPROVED** [review]

**Pull request merged**